bookbotproject/conversationtree/admin.py

```diff
@@ -18,7 +18,14 @@
                     list_display=('template', 'node',),
                     list_display_links=('template',)
                     ,)
+admin.site.register(Book,
+                    admin.ModelAdmin,
+                    list_filter=('author', ),
```

Comment:
As I understood, we need author field to search a book, If I am correct, I have some ideas,
Filtering books by only using author may not be sufficient. We may think to add a new field:"genre" to our structure, In addition to adding a new field, we can use this field as a filtering field in here. If I misunderstood anything, please correct me by opening a discussion in Issues.(This place may not be the best to have a discussion)

Reply:
I agree with you. I am adding the genre field now, since it will be nullable, we can choose not to use it in the worst case scenario. :)